api-reference/10 UI Components/BaseChart/7 Chart Elements/Label/3 Methods/shift(x_y).md

```diff
@@ -3,16 +3,141 @@ id: baseLabelObject.shift(x, y)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Moves label to the specified location.
 
 ##### return: Object
-<!-- Description goes here -->
+The label's instance.
 
 ##### param(x): Number
-<!-- Description goes here -->
+The `x` coordinate.
 
 ##### param(y): Number
-<!-- Description goes here -->
+The `y` coordinate.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The origin is at the upper left corner of the Chart. The coordinates represent the distance between a label and the origin and should be positive numbers.
+
+The following example moves label of a point of a bar Chart to the position above the bar:
```

Comment:
I want to change the part that repeats "of a" in close succession: "label of a point of a bar Chart". 

```suggestion
The following example customizes a Bar Chart. The code moves a label to the top of the corresponding bar. 
```